src/main/java/Miku.java

```diff
@@ -0,0 +1,126 @@
+import java.util.Scanner;
+
+public class Miku {
+
+    static Task[] storedList = new Task[100];
+    static int numberOfListItems = 0;
+
+    public static void newTodo(String newItem) throws MikuException {
+        String[] itemString = newItem.split("todo");
+        if (itemString.length < 1) {
+            throw new MikuException();
+        } else {
+            storedList[numberOfListItems] = new Todo(itemString[1]);
+            storedList[numberOfListItems].description = (itemString[1]);
+            System.out.println("Got it! I've added this task:\n" + storedList[numberOfListItems].toString());
+            numberOfListItems++;
+            System.out.println("Now you have " + numberOfListItems + " tasks in the list!");
+        }
+    }
+
+    public static void newDeadline(String newItem) throws MikuException {
+        String[] itemString = newItem.split("deadline|/by");
+        if (itemString.length < 1 || !newItem.contains("/by ")) {
+            throw new MikuException();
+        } else {
+            storedList[numberOfListItems] = new Deadline(itemString[1], itemString[2]);
+            storedList[numberOfListItems].description = (itemString[1]);
+            System.out.println("Got it! I've added this task:\n" + storedList[numberOfListItems].toString());
+            numberOfListItems++;
+            System.out.println("Now you have " + numberOfListItems + " tasks in the list!");
+        }
+    }
+
+    public static void newEvent(String newItem) throws MikuException {
+        String[] itemString = newItem.split("event|/from|/to");
+        if (itemString.length < 1 || !newItem.contains("/from ") || !newItem.contains("/to ")) {
+            throw new MikuException();
+        } else {
+            storedList[numberOfListItems] = new Event(itemString[1], itemString[2], itemString[3]);
+            storedList[numberOfListItems].description = (itemString[1]);
+            System.out.println("Got it! I've added this task:\n" + storedList[numberOfListItems].toString());
+            numberOfListItems++;
+            System.out.println("Now you have " + numberOfListItems + " tasks in the list!");
+        }
+    }
+
+
+    public static void main(String[] args) {
```

Comment:
You could consider refactoring parts of this method to separate methods to adhere to this code quality [guideline](https://nus-cs2113-ay2324s2.github.io/website/se-book-adapted/chapters/codeQuality.html#avoid-long-methods) and to avoid deep nesting ([guideline](https://nus-cs2113-ay2324s2.github.io/website/se-book-adapted/chapters/codeQuality.html#avoid-deep-nesting))